SQL/beestation_schema.sql

```diff
@@ -532,6 +532,18 @@ CREATE TABLE IF NOT EXISTS `SS13_stickyban_matched_ip` (
   PRIMARY KEY (`stickyban`,`matched_ip`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+--
```

Comment:
Is there anything we will need to handle with our own database before taking on their new achievement system?